.devcontainer/Dockerfile

```
@@ -1,11 +1,14 @@
-FROM node:21-alpine3.19
+FROM node:22-alpine
```

```suggestion
FROM node:22-alpine3.21
```

Just to make it obvious which alpine base image is used.